### DISCUSSION TITLE
Checkout Page

### DISCUSSION THREAD

**Discussion by ONLIXO** [woocommerce/woocommerce#48512] [Checkout Flow]
Hi,
On the "Checkout page", in a "Payment option" section, my name is showing instead of my store name. How can I fix that issue?

**Comment by nielslange** [comment]
Hello @ONLIXO 👋

Please share a screenshot that shows the corresponding section. This helps to provide an answer to your question.

Please also note that the GitHub discussions are more for technical discussions, such as implementing a new feature. The best place, to get support for WooCommerce is https://wordpress.org/support/plugin/woocommerce/. That said, if you share a screenshot of the issue you're facing, we should be able to assist as well.

**Reply by ONLIXO** [reply]
Hi @nielslange 
Thank you so much for your reply. Here is screenshot. 

![CheckOut](https://github.com/woocommerce/woocommerce/assets/172936979/5c8739bb-4d0d-4d35-8a0d-3f2b9890c166)

**Reply by nielslange** [reply]
Thank you for sharing the screenshot, @ONLIXO. WooCommerce does not support credit card payments by default, so the payment option shown in your screenshot likely originates from an extension. I recommend contacting the support team for that extension to resolve the issue of your name appearing instead of your company's.

Since there are no actionable items for us, I'll close this discussion. However, feel free to start a new one if you have more to discuss. For reporting bugs or suggesting features, please use the following link to create an issue: 

https://github.com/woocommerce/woocommerce/issues/new/choose

Discussions here should focus on gathering community feedback about potential changes.
